tests/policy/TestAccBasicAccessPolicyResource.tf

```diff
@@ -54,14 +77,39 @@ resource "aembit_access_policy" "first_policy" {
     client_workload = aembit_client_workload.first_client.id
     trust_providers = []
     access_conditions = []
-    credential_provider = aembit_credential_provider.api_key.id
+    credential_provider = aembit_credential_provider.snowflake1.id
     server_workload = aembit_server_workload.first_server.id
 }
 
 resource "aembit_access_policy" "second_policy" {
     is_active = false
     name = "TF Second Policy"
     client_workload = aembit_client_workload.second_client.id
-    credential_provider = aembit_credential_provider.api_key.id
+    credential_provider = aembit_credential_provider.snowflake2.id
     server_workload = aembit_server_workload.first_server.id
-}
\ No newline at end of file
+}
+
+resource "aembit_access_policy" "third_policy" {

```

Comment:
Please move the new multiple credential provider resource tests into a standalone tf/tfmod file set.